packages/react/src/components/AnimatedHeader/components/Tiles/BaseTile/BaseTile.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import { AIPromptTile, GlassTile } from '../index';
```

Comment:
I believe this is where you’re seeing the circular dependency since `BaseTile` imports from `index` but `index` also imports `BaseTile`.

```suggestion
import { AIPromptTile } from '../AIPromptTile/AIPromptTile';
import { GlassTile } from '../GlassTile/GlassTile';
```